### PR TITLE
Add Export command

### PR DESCRIPTION
**Pull request by ChrisHo1341**
Export command exports contacts currently shown to an external json file.

For example, if "find" command is used, "export" command can be used to save these contacrs.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ExportCommand.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    /**
+     * 
+     * @param indexes string of index
+     */
+    public ExportCommand() {}
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Person> lastShownList = model.getFilteredPersonList();
```
Comment (lihaoquan) [inline src/main/java/seedu/address/logic/commands/ExportCommand.java:37]: If we are using Applicant class instead, then might need to change this to Person @ys112

**Review by lihaoquan — COMMENTED** [review]
Generally LGTM. Added some queries to some of the lines.

File: src/main/java/seedu/address/logic/commands/ExportCommand.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+        JsonAddressBookStorage storage = new JsonAddressBookStorage(savePath);
+        AddressBook book = new AddressBook();
+        book.setPersons(lastShownList);
+        
+        try {
+            storage.saveAddressBook(book);
+        } catch (IOException e) {
+            System.out.println("life is liddat");
```
Comment (ys112) [inline src/main/java/seedu/address/logic/commands/ExportCommand.java:46]: image

Consider implementing something like this for this part by throwing command exception. Code can be found in LogicManager.java

**Review by ys112 — COMMENTED** [review]
Generally, code works. Just some details to note and check style issues like trailing whitespaces to fix. To check more details, please run gradlew clean build to verify in the console.

File: src/main/java/seedu/address/logic/commands/ExportCommand.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        }
```
[the remark this answers is not in the document]
Comment (ChrisHo1341) [reply src/main/java/seedu/address/logic/commands/ExportCommand.java:47]: Oh ya i forgot abt this, yea can do

[the remark this answers is not in the document]
Comment (ChrisHo1341) [reply src/main/java/seedu/address/logic/commands/ExportCommand.java:47]: Got it

File: src/main/java/seedu/address/logic/commands/ExportCommand.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class ExportCommand extends Command {
+    
+    public static final String COMMAND_WORD = "export";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
+    + "Example: " + COMMAND_WORD;
```
[the remark this answers is not in the document]
Comment (ChrisHo1341) [reply src/main/java/seedu/address/logic/commands/ExportCommand.java:21]: Will do sir, thank you

**Comment by xinen26** [comment]
Looks good to me! Good job.

**Review by xinen26 — APPROVED** [review]

**Pull request merged**